bip-0174.mediawiki

```diff
@@ -522,10 +506,14 @@ Any data types, their associated scope and BIP number must be defined here
 !BIP Number
 |-
 | Global
-| 0,1,2,3,4
+| 0
```

Comment:
Could this be written as a table that includes the name or one-line description of the field? I expect that needing looking things up across BIPs may end up being a pain.

Reply:
Done